bugrap/src/main/java/org/vaadin/bugrap/ui/LoginView.java

```diff
@@ -23,7 +22,7 @@ public LoginView(LoginModel m) {
 
 	@Override
 	public void enter(ViewChangeEvent event) {
-		if (BaseModel.loginUser != null)
+		if (model.getLoginUser() != null)
```

Comment:
This if is looking strange. Is it correct?

Reply:
it's actualy working fine, but I agree its looking kinda strange :)